modules/renter/upload.go

```diff
@@ -20,13 +20,31 @@ import (
 
 	"github.com/NebulousLabs/Sia/build"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/renter/siafile"
 )
 
 var (
 	// errUploadDirectory is returned if the user tries to upload a directory.
 	errUploadDirectory = errors.New("cannot upload directory")
 )
 
+// newFile is a helper to more easily create a new Siafile for testing.
```

Comment:
is this comment correct? Is this function in the right file? This isn't a testing file.